src/core/app_config/index.ts

```diff
@@ -52,8 +69,42 @@ export default (() => {
         url: 'https://www.openstreetmap.org/edit?editor=remote#map=',
       },
     ],
+    effectiveFeatures: {},
   };
-})();
+  configs['global'] = globalAppConfig;
+  return globalAppConfig;
+}
+
+function getEffectiveFeatures(appConfig: AppConfigEffective) {
+  return Object.fromEntries(
+    [...(appConfig.featuresByDefault ?? []), ...(appConfig.features ?? [])].map((k) => [
+      k,
+      true,
+    ]),
+  );
+}
+
+export function getEffectiveConfig(): AppConfigEffective {
```

Comment:
considering 'effective' naming, let me remind you to give us smth to read about using this word
this parcticular functiona has an effect
is that intended and is that why it's named effective? i suppose no, but who knows 👯‍♂️

Reply:
https://networkencyclopedia.com/effective-permissions/